'''
Exception classes

自定义异常
'''


class CommandException(Exception):
    '''未找到命令'''
    def __init__(self, arg):
        err = '[-] Unknown command：{0}'.format(arg)
        Exception.__init__(self, err)
        self.arg = arg


class ErrorException(Exception):
    '''错误信息'''
    def __init__(self, arg):
        err = '[!] Error：{0}'.format(arg)
        Exception.__init__(self, err)
        self.arg = arg


# def check_shuzi(num):
#     if num == 0:
#         return True
#     else:
#         return False
#
#
# try:
#     a = check_shuzi(1)
#     if a != True:
#         raise ErrorException('test')
# except ErrorException as e:
#     print_color_red(e)
